### ISSUE TITLE
MedChemica Compound Design

### ISSUE THREAD

**Issue by danaklug** [opensourceantibiotics/Series-2-Diarylimidazoles#14]
I was recently able to get access to a temporary license for MedChemica’s [MC Pairs software](https://www.medchemica.com/products/). In a nutshell, this software mines the literature for matched pairs and analyzes the effects of specific changes on parameters like solubility, metabolism, and activity against certain targets. It will also then suggest compounds to make based on your parent structure, the parameters you want to influence, and its matched pair analysis. 

I ran a search using **OSA_000822** looking for suggestions to improve clearance. The results are posted [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Misc%20Contributions/OSA_000822_Decrease-metabolism_RuleDesign_refined_public.xlsx). A quick guide to the spreadsheet: 

“**results**" tab: Shows suggested compounds and their likely effects on various clearance and other assays. (NED = not enough data). I have curated this tab at MedChemica’s request and have also removed the raw data from the spreadsheet, but I have permission to share the full results 1:1 (just not post publicly) if anyone is interested.
“**query inputs 1**” tab: The search parameters I used.
“**server messages 1**” tab: Any errors that came up during the search are catalogued.
“**starting materials**” tab: This is a tab I have added with CAS and Pubchem IDs for potential starting materials to make some of the suggested analogues, if we decide to pursue them (for future reference). 

Interestingly, most of the suggestions involve substituents on the pyridine ring as opposed to the benzothiophene. IMO it would be worth synthesizing a handful of these to probe this region of the molecule even though it’s not a predicted metabolic hotspot.

**Comment by mattodd** [comment]
Interesting. So there's only one prediction that is thought to reduce human microsomal clearance (row 3)?

**Comment by danaklug** [comment]
Yes, looks like it, although there are a lot of blanks or NEDs in that column so it seems the data just isn't there.

I'm really surprised that the software didn't pick up the benzothiophene as I'd imagined this to be a more common problem - although maybe people are just avoiding them in the first place so they don't have to fix it later - but it is nice to have a bit of guidance on what modifications to the pyridine may be beneficial.

**Comment by danaklug** [comment]
To give an idea of the proposed structures, here are some selected pyridine analogues which are predicted to decrease clearance in at least one of the assays (LM Clint or hep) in the "results" tab (parent structure highlighted in yellow).
![MCPairs_representative_analogues](https://user-images.githubusercontent.com/53487492/83750857-d5f51700-a65d-11ea-82c2-68ae9996aac9.PNG)

**Comment by mattodd** [comment]
Great. So for the above, would we be able to access these structures easily with commercially-available materials?

**Comment by danaklug** [comment]
For the most part, yes. Might be a bit steppy, but doable.
